src/index.js

```diff
@@ -1,10 +1,19 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import "./index.css";
-import App from "./App";
-import * as serviceWorker from "./serviceWorker";
+import React from 'react';
```

Comment:
Please import first 3rd party libraries alphabetically and then your files, and add an empty line between the two groups.